cli/asc.js

```diff
@@ -140,13 +140,19 @@ function loadAssemblyScript() {
     __pin = assemblyscript.__pin;
     __unpin = assemblyscript.__unpin;
     __collect = assemblyscript.__collect;
+    __wrapDiagnosticMessage = assemblyscript.DiagnosticMessage.wrap;
+    __wrapRange = assemblyscript.Range.wrap;
+    __wrapSource = assemblyscript.Source.wrap;
```

Comment:
Good idea to use the loader here. Makes me wonder if it'd make sense to simplify how this is used, perhaps a general helper along the lines of:

```js
function ensureObject(ptrOrObj, ctor) {
  if (typeof ptrOrObj === "number") return ctor.wrap(ptr);
  return ptrOrObj;
}
ensureObject(range, assemblyscript.Range);
```

Reply:
I've created `__wrap` function :)